src/webhook/index.js

```diff
@@ -7,7 +7,7 @@ import filters from './../filters';
 module.exports = ({
   parse: (service, pullRequestId, owner, repo, rawOutput) =>
     // Get list of files in a specific pull request
-    service.pulls.listFiles({ owner, repo, number: pullRequestId })
+    service.pulls.listFiles({ owner, repo, pull_number: pullRequestId })
```

Comment:
I can't find more context for that so I just need a verification it's an intended change

Reply:
This particular change comes from the deprecation warning reported by @octokit/rest library. They changed the parameter name from `number` to `pull_number` in the latest release.